### PR TITLE
sync block test

### PR DESCRIPTION
**Pull request by ququzone**
work for  #1535

### CODE REVIEW

File: test/sync/test.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+	_ "go.uber.org/automaxprocs"
+	"go.uber.org/zap"
+	glog "log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/iotexproject/iotex-core/config"
+	"github.com/iotexproject/iotex-core/pkg/log"
```
Comment (CoderZhi) [inline test/sync/test.go]: wrong grouping

File: test/sync/test.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+	stop := make(chan os.Signal, 1)
+	signal.Notify(stop, os.Interrupt)
+	signal.Notify(stop, syscall.SIGTERM)
+	ctx, cancel := context.WithCancel(context.Background())
+	stopped := make(chan struct{})
+	livenessCtx, livenessCancel := context.WithCancel(context.Background())
+
+	genesisCfg, err := genesis.New()
+	if err != nil {
+		glog.Fatalln("Failed to new genesis config.", zap.Error(err))
```
Comment (CoderZhi) [inline test/sync/test.go]: log.L().Fatal

File: test/sync/test.go [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+	}
+	start := meta.ChainMeta.Height
+
+	for {
+		time.Sleep(time.Minute * 5)
+		meta, err = c.API().GetChainMeta(context.Background(), &iotexapi.GetChainMetaRequest{})
+		if err != nil {
+			glog.Fatal(err)
+		}
+		if start >= meta.ChainMeta.Height {
```
Comment (CoderZhi) [inline test/sync/test.go]: start is not updated in the for loop. Therefore, if the first time it works, the if condition will always be true.

Comment (CoderZhi) [reply test/sync/test.go]:
```
	start := 0
	for {
		time.Sleep(time.Minute * 5)
		meta, err = c.API().GetChainMeta(context.Background(), &iotexapi.GetChainMetaRequest{})
		if err != nil {
			log.L().Fatal(err)
		}
		if start >= meta.ChainMeta.Height {
			glog.Fatalf("blockchain stop sync at %d height", start)
		}
                start = meta.ChainMeta.Height
	}
```

**Comment by yutongp** [comment]
we achieved this feature through other channel. so close this pr

**Pull request closed without merging**